Mathlib/CategoryTheory/Monoidal/Rigid/Braided.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2024 Gareth Ma. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Gareth Ma
+-/
+import Mathlib.CategoryTheory.Monoidal.Rigid.Basic
+import Mathlib.CategoryTheory.Monoidal.Braided.Basic
+
+/-!
+# Mirror rigid structures via a braiding equivalence.
```

Comment:
Don't like the title. "braiding equivalence" is referring to a braiding-preserving equivalence of categories, which is not what you mean here.